packages/main/src/plugin/dockerode/libpod-dockerode.ts

```diff
@@ -72,9 +72,35 @@ export interface PlayKubeInfo {
   Volumes: { Name: string }[];
 }
 
+export interface PodCreatePortOptions {
+  host_ip: string;
```

Comment:
Shouldn't all of this be camelCase instead of snake_case ? I find that a lot of code switches between the two for `podman-desktop`.

I can't find a style guide for TypeScript, but we should stick to camelCase or snake_case throughout podman-desktop